Chat/ClientApp/src/components/ConfigurationScreen.tsx

```diff
@@ -37,7 +37,8 @@ import { MAXIMUM_LENGTH_OF_NAME } from '../constants';
 
 export interface ConfigurationScreenProps {
   joinChatHandler(): void;
-  setup(displayName: string, emoji: string, joinChatHandler: Function): void;
+  endChatHandler(): void;
```

Comment:
similar to comment above its really that you are removed from the chat and you need to handle it